src/main/resources/application.properties

```diff
@@ -7,3 +7,5 @@ storage.password=password
 #in seconds, set to 0 to disable
 storage.expiry=0
 logging.level.org.springframework.web.filter.CommonsRequestLoggingFilter=DEBUG
+springdoc.swagger-ui.url=/swagger.json
+springdoc.swagger-ui.path=/apidocs
```

Comment:
wow, so just these 2 config files and it all JustWorks?

Reply:
yeah the application properties from the swagger library allow us to override the default url paths quite easily.